src/ctapipe/io/datawriter.py

```diff
@@ -44,8 +44,10 @@ def _get_tel_index(event, tel_id):
 #   (meaning readers need to update scripts)
 # - increase the minor number if new columns or datasets are added
 # - increase the patch number if there is a small bugfix to the model.
-DATA_MODEL_VERSION = "v7.1.0"
+DATA_MODEL_VERSION = "v7.2.0"
```

Comment:
You would need to update the compatible data model version and some unittests. See changes in #2901